### PR TITLE
Minor amends to programming languages page

### PR DESCRIPTION
**Pull request by louzoid-gds**
Renamed Sign In to One Login
Removed reference to Verify Passport
All other content looks accurate

### CODE REVIEW

File: source/standards/programming-languages.html.md.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Programming languages
-last_reviewed_on: 2022-07-14
+last_reviewed_on: 2022-01-02
```
Comment (domoscargin) [inline source/standards/programming-languages.html.md.erb:3]:
```suggestion
last_reviewed_on: 2023-03-01
```

Comment (louzoid-gds) [reply source/standards/programming-languages.html.md.erb]: Doh! Thank you. Corrected

**Review by domoscargin — APPROVED** [review]
Looks good!

File: source/standards/programming-languages.html.md.erb (hunk 1 above)
Comment (louzoid-gds) [reply source/standards/programming-languages.html.md.erb]: Ha! No it's still wrong. Says 2022. I think I need to go have a lie down somewhere

**Review by domoscargin — APPROVED** [review]

**Pull request merged**